core/Tracker/Cache.php

```diff
@@ -119,6 +119,14 @@ public static function clearCacheGeneral()
     }
 
     /**
+     * Clear website cache.
+     */
+    public static function clearWebsiteCache($idSite)
+    {
+        self::getCache()->delete((int) $idSite);
+    }
```

Comment:
Will remove that one. Didn't find it as the other method was called `clearCacheGeneral` but this one `delete...` (inconsistent naming)